src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -68,17 +72,30 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
 		>
 			{delegate?.dataSource?.length > 1 ? (
 				<div
-					className={`ml-[-0.6px] mr-[-0.6px] mt-[-0.5px] flex h-8 items-center space-x-3 rounded-t-md border-[1px] border-solid px-5 ${
+					className={`ml-[-0.6px] mr-[-0.6px] mt-[-0.5px] flex h-8 items-center space-x-[7px] rounded-t-md border-[1px] border-solid px-3 sm:space-x-3 sm:px-5 ${
 						delegate?.dataSource.length > 1
 							? 'border-[#485F7D] bg-[#EDEFF3] dark:border-[#9E9E9E] dark:bg-[#3D3F41]'
 							: 'border-[#F89118] bg-[#FFF7EF] dark:border-[#F89118] dark:bg-[#422A0D]'
 					} `}
 				>
 					{!!delegate?.dataSource?.includes(EDelegateSource.POLKASSEMBLY) && (
-						<div className='flex items-center space-x-3'>
+						<div className='flex items-center space-x-2 sm:space-x-3'>
 							<div className='flex items-center space-x-1'>
-								<PolkadotIcon />
-								<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Polkassembly</span>
+								<Image
+									src={'/assets/delegation-tracks/pa-logo-small-delegate.svg'}
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image